### ISSUE TITLE
Open SQL file path

### ISSUE THREAD

**Issue by eevleevs** [sqlitebrowser/sqlitebrowser#1014]
Open SQL file dialog should default to the same path as the database; instead it opens home folder (tested on Windows).

**Comment by justinclift** [comment]
@eevleevs As a thought, there is an option in the Preferences dialog for adjusting this.

Saying that because over the years we've had people with distinctly different needs for the "default location to open" open issues here, so we made it configurable.

Are you ok to look through there and see if the current options suit? :smile:

**Comment by eevleevs** [comment]
Suit only if you work on a single directory. If you use multiple databases in different places it is still uncomfortable. I would favor an option to open the directory where the active database is.

**Comment by justinclift** [comment]
Yep, that sounds reasonable. :smile:

**Comment by smile-on** [comment]
Are you open for a patch to make this change?

**Comment by justinclift** [comment]
Yep, we're pretty open with things as long as the code quality is reasonable. :smile:
